pkg/podimpersonation/podimpersonation.go

```diff
@@ -559,9 +568,33 @@ func (s *PodImpersonation) augmentPod(pod *v1.Pod, sa *v1.ServiceAccount, secret
 				continue
 			}
 
+			vmount := v1.VolumeMount{
+				Name:      "user-kubeconfig",
+				MountPath: "/tmp/.kube",
+			}
+			cfgVMount := v1.VolumeMount{
+				Name:      "user-kubeconfig-map",
+				MountPath: "/home/.kube/config",
+				SubPath:   "config",
+			}
+
+			pod.Spec.InitContainers = []v1.Container{
+				{
+					Name:         "init-container",
+					Image:        "busybox",
```

Comment:
Should we use our own busybox ? rancher/mirrored-bci-busybox or rancher/library-busybox. I don't know which is the latest one. Maybe ask @pjbgf ?

Reply:
`registry.suse.com/bci/bci-busybox`.

Reply:
Good call, done